rustfmt.toml

```diff
@@ -1,4 +1,4 @@
-match_block_trailing_comma = true
+match_block_trailing_comma = false
```

Comment:
Why did you change this?

Reply:
`match_block_trailing_comma` is unstable feature which is only available on nightly channel.
When cargo fmt is run on stable channel, the job fails - https://travis-ci.org/github/nox/serde_urlencoded/jobs/715205245#L237

I have configured to run rustfmt to be run only on rust stable since rustfmt is sometimes not available on latest nightly channel.

Reply:
Thanks for the explanation.